src/components/vouchers/VoucherList.tsx

```diff
@@ -115,13 +125,15 @@ export const VoucherList = ({
       estimatedItemSize={MAX_INITIAL_NUM_TO_RENDER}
       keyExtractor={keyExtractor}
       ListFooterComponent={() => {
-        if (data?.length >= MAX_INITIAL_NUM_TO_RENDER) {
+        if (sectionedData?.length >= MAX_INITIAL_NUM_TO_RENDER) {
```

Comment:
this just works because of FlashList right? otherwise with SectionList we could not count the object

Reply:
but in sectioned data the section items are included in the length, which distorts the data count?

Reply:
i reverted to data to count the real data amount